deploy/roles/full-app/templates/config/dango/app.toml

```diff
@@ -19,6 +19,13 @@ wasm_cache_capacity = 1000
 # Gas limit when serving query requests.
 query_gas_limit = 100000000
 
+# Range of database keys that will be loaded into memory for better performance.
+# Lower bound is inclusive, upper bound is exclusive.
+priority_range = [
+  "7761736d8dd37b7e12d36bbe1c00ce9f0c341bfe1712e73f", # = b"wasm" + dex_contract_address
+  "7761736d8dd37b7e12d36bbe1c00ce9f0c341bfe1712e740", # = b"wasm" + increment_last_byte(dex_contract_address)
+]
```

Comment:
@penso I'm adding this new field in the `app.toml`. Can you confirm if I do `just deploy-testnet`, is this applied to the testnet?